framework/error.go

```diff
@@ -18,6 +21,19 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
+var invalidResourceVersionError = errors.StatusError{
+	ErrStatus: metav1.Status{
+		Status:  "Failure",
+		Code:    500,
+		Message: "Testing value /metadata/resourceVersion failed",
+	},
+}
+
+// IsInvalidResourceVersionError asserts invalidResourceVersionError.
+func IsInvalidResourceVersionError(err error) bool {
+	return err == error(&invalidResourceVersionError)
```

Comment:
I think this should be checked against the cause. Like `err` -> `microerror.Cause(err)`. Unit tests might be useful here when we are already in doubt.

Reply:
This actually added 0 value anymore after I started to always get the latest resource with retries. I removed it.